portfolio/src/main/webapp/contact.html

```diff
@@ -23,28 +25,33 @@
 
 	<div class="contact-me">
 		<h1>Contact Me!</h1>
-		<form action="/data" method="POST">
+		<form id="comment-form" method="POST" enctype="multipart/form-data">
 			<div class="form-content">
 				<label for="name">Name</label>
 				<input type="text" name="name" id="name" placeholder="optional"><br>
-            </div>
+      		</div>
+			<div class="form-content">
+				<label for="name">Upload an image</label>
+				<input type="file" name="image"><br>
+			</div>
 			<div class="form-content">
-                <label for="comment">Comments</label>
-                <textarea id="comment" name="comment" rows="7" cols="30" placeholder="Type your comments here!" required></textarea><br>
-            </div>
-            <button role="submit" name="commentButton">Submit</button>
+				<label for="comment">Comments</label>
```

Comment:
nit: indent to be in-line with previous div

Reply:
> nit: indent to be in-line with previous div

Seems like I cannot do it on github, I will update in the next repo